backend/src/main/java/com/tcs/dhv/entity/Parcel.java

```diff
@@ -0,0 +1,84 @@
+package com.tcs.dhv.entity;
+
+import com.tcs.dhv.enums.DeliveryType;
+import com.tcs.dhv.enums.ParcelStatus;
+import com.tcs.dhv.enums.PaymentType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.NotNull;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+import java.time.OffsetDateTime;
+import java.util.UUID;
+
+@Entity
+@NoArgsConstructor
+@Getter
+@Setter
+@ToString
+@Table(name = "parcels")
+public class Parcel {
+
+    @Id
+    @Setter(AccessLevel.NONE)
+    @GeneratedValue(strategy = GenerationType.UUID)
+    private UUID id;
+
+    @NotNull
+    @ManyToOne
+    @JoinColumn(name = "sender_id")
+    private User sender;
+
+    @NotNull
+    @ManyToOne
+    @JoinColumn(name = "recipient_address_id")
+    private Address recipientAddress;
```

Comment:
```suggestion
    @NotNull
    @JoinColumn(name = "sender_id")
    private User sender;

    @NotNull
    @JoinColumn(name = "recipient_address_id")
    private Address recipientAddress;
```

A parcel has only one recipient address and one sender